config/environments/development.rb

```diff
@@ -57,7 +57,7 @@
     }
 
     # These numbers are from the recommended dalli config: https://devcenter.heroku.com/articles/memcachier#ruby
-    config.cache_store = :dalli_store, nil, {
+    config.cache_store = :memory_store, nil, {
```

Comment:
If we want to change this to use `:memory_store`, then we should not pass any additional arguments.
```ruby
 config.cache_store = :memory_store
```
vs
```ruby
 config.cache_store = :memory_store, nil, {...}
```

Reply:
Ah good catch, that change persisted from my local env, didn't intend to push that up yet. Changing back for now.